script/hassfest/__main__.py

```diff
@@ -84,6 +86,7 @@ def get_config() -> Config:
     parser.add_argument(
         "--integration-path",
         action="append",
+        dest="specific_integrations",
```

Comment:
I kind of understand why you want to rename the destination, but I'm not convinced by this change.
IMO it is easier to understand if the argument name matches the property name.

I would park it for now (or move to a separate PR if you wish a specific review)

Reply:
I think it would be more confusing to type a singular name to be `list[str] | None`. This change is entirely confined into this function - I don't see much of an issue...

Reply:
I'm not sure why you are referring to `list[str] | None`

What I am saying is that I think instead of adjusting the destination, the argument should probably be renamed, and then it becomes an even bigger change.
Until then, I don't see an issue with `specific_integrations=parsed.integration_path`

Reply:
> I'm not sure why you are referring to `list[str] | None`

Sorry, I meant `list[pathlib.Path] | None`, the type of `hassfest.models.Config.specific_integrations`.

> Until then, I don't see an issue with `specific_integrations=parsed.integration_path`

It reads pretty confusingly, don't you think? It looks like you'd be assigning a single path from `parsed` to what is a plural field.

Reply:
I agree it's confusing - I just feel that it should be resolved in a separate PR.
Having `dest=` doesn't really resolve the confusion.

Reply:
I think it's a small enough change (and internal to this function, as discussed above), that it really doesn't hurt here (if e.g. removing an unused argument in the `requirements` typing PR is okay too :) )?

Reply:
> if e.g. removing an unused argument in the `requirements` typing PR is okay too

I explained in the PR => the definition was touched anyway.
In this case, you are changing the function content without needing to (the type hints are still valid with the old variable name), and it is a change that I feel is wrong (or at least incomplete).

Reply:
And here I'm touching the definition of `Config` to make it actually correct re what's being put in there from the untyped `parsed` bag...

Reply:
I think this is confusing and an unneeded change. Additionally, it has nothing to do with typing at all. Please revert it.

Reply:
Reverted.